main.js

```diff
@@ -5,6 +5,37 @@ const close = document.querySelector('.close-icon');
 const overlayLinkOne = document.querySelector('.overlay-link-one');
 const overlayLinkTwo = document.querySelector('.overlay-link-two');
 const overlayLinkThree = document.querySelector('.overlay-link-three');
+const desktopWorkSection = document.getElementById('desktop-works');
+const mobileModalButtonOne = document.querySelector('#bttn');
+const mobileModalButtonTwo = document.querySelector('.btn-touch-two');
+const mobileModalButtonThree = document.querySelector('.btn-touch-three');
+const mobileModalButtonFour = document.querySelector('.btn-touch-four');
+const closeMobileButton = document.querySelector('.close-mobile-button');
+
+mobileModalButtonOne.addEventListener('click', () => {
+  document.querySelector('.desktop-mode').style.display = 'none';
+  document.querySelector('.mobile-modal-container').style.display = 'block';
+});
+
+mobileModalButtonTwo.addEventListener('click', () => {
```

Comment:
- hi @OyePriscilla Good job implementing the project thus far, Please on the desktop view the dynamic display of projects does not work as it is empty, please try fixing it. 

##### Your design 

https://user-images.githubusercontent.com/58771507/174150151-3b1cb869-2308-4f87-bfed-89f441e7afac.mp4